src/fpm.f90

```diff
@@ -467,6 +469,13 @@ subroutine cmd_build(settings)
     call build_package(targets,model,verbose=settings%verbose)
 endif
 
+do i=1, size(targets)
+    if (targets(i)%ptr%target_type == FPM_TARGET_OBJECT .and. package%build%shared_library) then
```

Comment:
Is there a direct way to get the access to build directory of ``FPM_TARGET_OBJECT`` ? @awvwgk

Reply:
Best check how the static library is built